### PR TITLE
Fix some bugs pointed out by SonarCloud

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/main/java/de/sub/goobi/config/ConfigCore.java [hunk 1]
```diff
@@ -33,15 +33,7 @@
...
-
-    /**
-     * Get the path for the temporary images directory.
-     *
-     * @return the path for the temporary images directory as String
-     */
-    public static String getTempImagesPath() {
-        return "/pages/imagesTemp/";
-    }
+    public static String imagesTemp = "/pages/imagesTemp/";
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/config/ConfigCore.java]: Why are you replacing this? Class member variables should never accessed directly as a public member variable. Or should this be a constant value?

File: Kitodo/src/main/java/de/sub/goobi/export/download/Multipage.java [hunk 2]
```diff
@@ -108,30 +108,21 @@ public void startExport(Process process) throws IOException {
             response.setHeader("Content-Disposition", "attachment;filename=\"" + fileName + "\"");
             ServletOutputStream out = response.getOutputStream();
 
-            /*
-             * die txt-Datei direkt in den Stream schreiben lassen
-             */
+            // die txt-Datei direkt in den Stream schreiben lassen
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/export/download/Multipage.java]: As you changing the comment, please translate this comment to English.

File: Kitodo/src/main/java/de/sub/goobi/export/download/Multipage.java [hunk 3]
```diff
@@ -108,30 +108,21 @@ public void startExport(Process process) throws IOException {
...
-            } finally {
-                if (fis != null) {
-                    fis.close();
-                }
             }
 
-            /*
-             * den Stream zurückgeben
-             */
+            // den Stream zurückgeben
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/export/download/Multipage.java]: As you changing the comment, please translate this comment to English.

File: Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java [hunk 4]
```diff
@@ -435,14 +435,12 @@ public String uploadFromHomeAlle() throws NumberFormatException, DataException,
...
-                    .substring(element.toString().indexOf("[") + 1, element.toString().indexOf("]")).trim();
+            String id = element.toString()
+                    .substring(element.toString().indexOf('[') + 1, element.toString().indexOf(']')).trim();
 
             for (Task step : (Iterable<Task>) this.page.getCompleteList()) {
-                /*
-                 * nur wenn der Schritt bereits im Bearbeitungsmodus ist, abschliessen
-                 */
-                if (step.getProcess().getId() == Integer.parseInt(myID)
+                // nur wenn der Schritt bereits im Bearbeitungsmodus ist, abschliessen
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java]: As you changing the comment, please translate this comment to English.

File: Kitodo/src/main/java/org/kitodo/services/data/LdapServerService.java [hunk 5]
```diff
@@ -540,44 +522,23 @@ public boolean changeUserPassword(User user, String inNewPassword) throws NoSuch
                 md.update(inNewPassword.getBytes(StandardCharsets.UTF_8));
                 String encryptedPassword = new String(Base64.encodeBase64(md.digest()), StandardCharsets.UTF_8);
 
-                /*
-                 * UserPasswort-Attribut ändern
-                 */
-                BasicAttribute userpassword = new BasicAttribute("userPassword",
+                // UserPasswort-Attribut ändern
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/services/data/LdapServerService.java]: As you changing the comment, please translate this comment to English.

File: Kitodo/src/main/java/org/kitodo/services/data/LdapServerService.java [hunk 6]
```diff
@@ -540,44 +522,23 @@ public boolean changeUserPassword(User user, String inNewPassword) throws NoSuch
...
-                BasicAttribute lanmgrpassword = null;
-                try {
-                    lanmgrpassword = new BasicAttribute("sambaLMPassword",
-                            LdapUser.toHexString(LdapUser.lmHash(inNewPassword)));
-                    // TODO: Don't catch super class exception, make sure that
-                    // the password isn't logged here
-                } catch (Exception e) {
-                    logger.error(e);
-                }
+                // LanMgr-Passwort-Attribut ändern
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/services/data/LdapServerService.java]: As you changing the comment, please translate this comment to English.

File: Kitodo/src/main/java/org/kitodo/services/data/LdapServerService.java [hunk 7]
```diff
@@ -540,44 +522,23 @@ public boolean changeUserPassword(User user, String inNewPassword) throws NoSuch
...
-                 */
-                BasicAttribute ntlmpassword = null;
-                try {
-                    byte[] hmm = digester.digest(inNewPassword.getBytes("UnicodeLittleUnmarked"));
-                    ntlmpassword = new BasicAttribute("sambaNTPassword", LdapUser.toHexString(hmm));
-                } catch (UnsupportedEncodingException e) {
-                    // TODO: Make sure that the password isn't logged here
-                    logger.error(e);
-                }
+                // NTLM-Passwort-Attribut ändern
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/services/data/LdapServerService.java]: As you changing the comment, please translate this comment to English.

File: Kitodo/src/main/java/de/sub/goobi/config/ConfigCore.java (hunk 1 above)
Comment (Beacze) [reply Kitodo/src/main/java/de/sub/goobi/config/ConfigCore.java]: Constant.

**Comment by henning-gerhardt** [comment]
There is a merge conflict in `Kitodo/src/main/java/de/sub/goobi/importer/ImportZentralblatt.java` - please fix this conflict while fixing the outstanding issues.

File: Kitodo/src/main/java/de/sub/goobi/config/ConfigCore.java (hunk 1 above)
Comment (henning-gerhardt) [reply Kitodo/src/main/java/de/sub/goobi/config/ConfigCore.java]: Then please change it to a constant according our coding and checkstyle rules.

**Pull request merged**